config.toml

```diff
@@ -110,9 +110,32 @@ weight = 3
 [Languages.en]
 languageName = "En"
 languageCode = "en-us"
-contentDir = "content/en"
 weight = 1
 
+## Mounts configuration
```

Comment:
In v0.56.0 Hugo introduced the mounts configuration. Documentation previously recommended storing assets likes images and scripts to the static directory. After that change, Hugo recommended stories assets in an assets directory. Instead of moving our assets, the mount configuration has been set so that the Hugo may get assets from the static directory to work with Image Processing.

The mount configuration also replaces `contentDir`. More info may be found [here](https://gohugo.io/hugo-modules/configuration/#module-configuration-mounts).

Reply:
Thank you for keeping us up to date!